utils/vpr_pbtype_to_eblif.py

```diff
@@ -85,15 +85,18 @@ def main(args):
         args.pb_type
     )
 
-    pbtype_name, clocks, inputs, outputs = ports(pbtype_leaf)
+    # FIXME: Ignore pbtype_leaf values connected to carry chains
```

Comment:
This particular change seems totally fine, maybe just hoist into an indepenent PR?